dynawo/sources/API/JOB/JOBJobEntry.h

```diff
@@ -97,6 +97,24 @@ class JobEntry {
    */
   const std::string& getName() const;
 
+  /**
+   * @brief Default constructor
+   */
+  JobEntry();
+
+  /**
+   * @brief Copy assignment operator
+   * @param other the job entry to copy
+   * @returns reference to this
+   */
+  JobEntry& operator=(const JobEntry& other);
```

Comment:
Unit test are missing for all those new operators

Reply:
OK